v2/devices/enchilada.yml

```diff
@@ -0,0 +1,121 @@
+name: "OnePlus 6/6T"
+codename: "enchilada"
+formfactor: "phone"
+aliases: []
+doppelgangers: []
+user_actions:
+  bootloader:
+    title: "Reboot to Bootloader"
+    description: "With the device powered off, press and hold the volume up, volume down and power buttons until the screen turns on. It should say 'Fastboot Mode' underneath the warning sign."
+    image: "phone_power_down"
+    button: true
+  recovery:
+    title: "Reboot to Recovery"
+    description: "With the device still at the 'Fastboot Mode' screen (if not, power off the device and press and hold volume down and power buttons until the screen turns on), select the 'Recovery Mode' option by using the volume buttons and press the power button to enter."
+    image: "phone_power_down"
+    button: true
+  downgrade_android:
+    title: "Downgrade Android"
+    description: "If the device is running an Android version above 9.0, you have to downgrade to the last 9.0 release. Please make sure to flash OxygenOS 9 before proceeding."
```

Comment:
description: "For ensured compatibility you should install Lineage 16.0 to the desired slot before flashing Ubuntu Touch. Other Android 9.0 based OS's may be compatible but not 10.0 or above"